components/radioButton/radioButton.js

```diff
@@ -0,0 +1,94 @@
+// Imports
+import React, { PropTypes } from 'react';
+
+import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
+
+function RadioButton({
+  checked,
+  children,
+  dataAttrs = false,
+  disabled,
+  id,
+  mods = [],
+  name,
+  onChange }) {
+  const restProps = getDataAttributes(dataAttrs);
+
+  if (disabled) {
+    mods.push('disabled_true');
+  }
+
+  const className = getClassNamesWithMods('ui-radio', mods);
+
+  return (
+    <div className={className} {...restProps}>
+      <input
+        checked={checked}
+        disabled={disabled}
+        id={id}
+        name={name}
+        onChange={onChange}
+        type="radio"
+      />
+      <label aria-checked={checked} htmlFor={id} role="radio">
```

Comment:
Here we are using the label to also style the input element. which means we are loosing the accessibility properties from the native input element. In this case we need to add this props so that we can navigate as we have the native element (ex.: navigating with tabs and focusing the element).